admin/src/app/foms/summary/summary.component.ts

```diff
@@ -44,7 +45,7 @@ import { AttachmentResolverSvc } from '@admin-core/services/AttachmentResolverSv
     styleUrls: ['./summary.component.scss']
 })
 export class SummaryComponent implements OnInit, OnDestroy {
-
+readonly projectPlanCodeEnum = ProjectPlanCodeEnum;
```

Comment:
Hi Ian, are we missing some space there for the formatting?

Reply:
Thanks for noticing that Catherine! Should be good now.